src/utils/messageTimestamp.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * createdAt is in UTC. Convert to local time using toLocaleString.
+ */
+export function formatCreatedAtToAMPM(createdAt: number) {
+  const date: Date = new Date(createdAt);
+
+  const options: Intl.DateTimeFormatOptions = {
+    hour: 'numeric',
+    minute: 'numeric',
+    hour12: true, // ensures the time is formatted in 12-hour clock notation with AM/PM.
+  };
+  const locale = navigator.language;
+  const localTimeString: string = date.toLocaleTimeString(locale, options);
```

Comment:
```suggestion
  const localTimeString = date.toLocaleTimeString(locale, options);
```
I think this should be fine without this since the return type of toLocaleTimeString is already string.

Reply:
updated to
```
  const time = format(createdAt || 0, 'p', { locale });

```